cmd/version.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dapr/cli/pkg/standalone"
+	"github.com/spf13/cobra"
+)
+
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print Dapr Runtime and Cli's version.",
+	Example: `
+# Version for Dapr
+	dapr version --output json
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		runtimeVersion := strings.Replace(standalone.GetRuntimeVersion(), "\n", "", -1)
+		if strings.Contains(outputInJSON, "json") {
+			fmt.Printf("{\"CLI version\": \"%s\",\"Runtime version\": \"%s\"}", cliVersion, runtimeVersion)
```

Comment:
It might be beneficial to define a simple version struct and rely on struct tags and marshalling to produce the representation to avoid any formatting errors, something like https://play.golang.com/p/J8tG_Q9F76S

Reply:
+1 to @jjcollinge 's suggestion.